code/src/index.css

```diff
@@ -5,9 +5,170 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: "Press Start 2P";
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-image: url("https://media.giphy.com/media/xT8qBfjJhOmNPTVWU0/giphy.gif");
+}
+
+/* START SCREEN */
+
+.nes-container.with-title.is-centered {
+  width: 50%;
+  min-height: 450px;
+  background: #f3abba;
+}
+
+.nes-container.with-title.is-centered > .title {
+  font-size: 1.7rem;
+  border: 3px solid black;
+  margin-top: -3rem;
+}
+
+.nes-input {
+  margin: 15px 0;
+}
+
+/* GAME SCREEN */
+
+.nes-container.game-container {
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  text-align: center;
+  background: #f58729;
+  padding: 30px;
+}
+
+.nes-container.game-container.end-container {
+  background-color: #fff;
+}
+
+.instruction-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  height: 45%;
+  margin-bottom: 30px;
+}
+
+.option-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  align-self: flex-end;
+  width: 45%;
+}
+
+.option-container p {
+  font-size: 12px;
+}
+
+.options-wrapper {
+  display: flex;
+  flex-direction: row;
```

Comment:
For readability purposes, one suggestion would be to set flex-direction to column when displaying the direction alternatives in mobile view. It looks a bit cramped 😀